### PR TITLE
core(aspect-ratio): skip aspect ratio audit for svg

### PR DESCRIPTION
**Pull request by peterjanes**
I'm pretty confident that the code matches the suggestion in #3716. I made an attempt at a new test, but it isn't correct (passes with and without the new check); suggestions welcome.

### CODE REVIEW

**Comment by peterjanes** [comment]
I've added my GitHub email to my Google account.

File: lighthouse-core/test/audits/image-aspect-ratio-test.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ describe('Images: aspect-ratio audit', () => {
           generateImage(
             {width: data.clientSize[0], height: data.clientSize[1]},
             {naturalWidth: data.naturalSize[0], naturalHeight: data.naturalSize[1]},
-            generateRecord(),
+            generateRecord({mimeType: data.mimeType}),
```
Comment (patrickhulce) [inline lighthouse-core/test/audits/image-aspect-ratio-test.js]: it's expecting two parameters so I believe this should look like `generateRecord(undefined, data.mimeType)`

then again, it doesn't really look like `generateRecord` is very useful for this test at all and was copied from another file so you can probably just ditch the function entirely and pass in the object directly to `generateImage`

File: lighthouse-core/audits/image-aspect-ratio.js [hunk 2]
```diff
@@ -73,6 +73,7 @@ class ImageAspectRatio extends Audit {
       // filter out images that don't have following properties
       // networkRecord, width, height, images that use `object-fit`: `cover` or `contain`
       return image.networkRecord &&
+        image.networkRecord.mimeType !== 'image/svg+xml' &&
```
Comment (patrickhulce) [inline lighthouse-core/audits/image-aspect-ratio.js]: want to update the comment real quick to explain svg's don't have valid natural dimensions :)

**Comment by paulirish** [comment]
huge! thanks so much @peterjanes  😁

**Comment by piotr-cz** [comment]
Does it fix inline SVG images?
I'm on Chrome Version 63.0.3239.108 (Official Build) (64-bit)

**Comment by patrickhulce** [comment]
@piotr-cz this PR won't show up in Chrome until M65 unfortunately

**Comment by piotr-cz** [comment]
No, I don't but it should be easy to reproduce with just any inline SVG image

**Comment by patrickhulce** [comment]
@piotr-cz it shouldn't be possible, and I'm not able to reproduce with any inline SVG image. That's why I'm asking if you remember where you saw it :)

**Comment by piotr-cz** [comment]
@patrickhulce The app I'm working on is quite complex and it would pretty hard to create test case out of it. It's not public, but I can send you a link by email to investigate if that's ok.

**Comment by patrickhulce** [comment]
sure that'd be great, email should be available on my GitHub profile

**Comment by piotr-cz** [comment]
@patrickhulce I've just send the link to email at Your GitHub profile.

Would be helpful, if Lighthouse would add more info, for example which image doesn't pass the test.

**Comment by patrickhulce** [comment]
@piotr-cz thanks for sending, I see the confusion now. You're referring to a different bug, #3551, which was fixed in 2.6 and should also be in Chrome 65 (use Chrome Canary, CLI, or extension for immediate workaround).

**Pull request merged**